pandas/tests/tslibs/test_parsing.py

```diff
@@ -284,10 +284,10 @@ def test_parse_time_string_check_instance_type_raise_exception():
         ("%Y%m%d %H:%M:%S", True),
         ("%Y-%m-%dT%H:%M:%S", True),
         ("%Y-%m-%dT%H:%M:%S%z", True),
-        ("%Y-%m-%dT%H:%M:%S%Z", True),
+        ("%Y-%m-%dT%H:%M:%S%Z", False),
```

Comment:
these are no longer detected as ISO because the pandas ISO8601 fastpath can't handle them